docs/operator-manual/ingress.md

```diff
@@ -58,6 +58,8 @@ metadata:
     kubernetes.io/ingress.class: nginx
     kubernetes.io/tls-acme: "true"
     nginx.ingress.kubernetes.io/ssl-passthrough: "true"
+    # If you face to a redirect loop. You need to enforce your nginx ingress use HTTPS protocol connect to backend. 
```

Comment:
Suggested rewording: If you encounter a redirect loop or are getting a 307 response code then you need to force the nginx ingress to connect to the backend using HTTPS.

Reply:
Done

Reply:
Thanks!